processors/github/github_processor_phase1_per_repo.py

```
@@ -0,0 +1,721 @@
+            chunks.append({
+                'text': '\n'.join(current_chunk),
+                'start': len(lines) - len(current_chunk),
+                'end': len(lines),
+                'type': 'code_block',
+                'file_path': file_path,
+                'language': language
+            })
+        
+        return chunks
+    
+    def _generate_embeddings_batch(self, chunks: List[Dict]):
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)